torchvision/ops/poolers.py

```diff
@@ -7,6 +7,22 @@
 from torchvision.ops.boxes import box_area
 
 
+def merge_levels_onnx(levels, unmerged_results):
+    first_result = unmerged_results[0]
+    dtype, device = first_result.dtype, first_result.device
+    res = torch.zeros((levels.size(0), first_result.size(1),
+                       first_result.size(2), first_result.size(3)),
+                      dtype=dtype, device=device)
+    for l in range(len(unmerged_results)):
+        index = (levels == l).nonzero().view(-1, 1, 1, 1)
+        index = index.expand(index.size(0),
+                             unmerged_results[l].size(1),
+                             unmerged_results[l].size(2),
+                             unmerged_results[l].size(3)).to(torch.long)
```

Comment:
do we need the `to(torch.long)` call? The `nonzero()` call always returns a `int64` type.